### PR TITLE
Fix utils.py: remove duplicate functions and merge changes

### PR DESCRIPTION
**Pull request by julian-pani**
Hi @ThilinaRajapakse 
Thanks for this repo!

The colab quickstart was broken, looks like there was a bad merge somewhere. See comments inline

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -226,63 +225,12 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
...
-    else:
-        raise KeyError(output_mode)
-
-    return InputFeatures(input_ids=input_ids,
-                        input_mask=input_mask,
-                        segment_ids=segment_ids,
-                        label_id=label_id)
-    
-
-def convert_examples_to_features(examples, label_list, max_seq_length,
```
Comment (julian-pani) [inline utils.py]: This function was duplicated, see above

File: utils.py [hunk 2]
```diff
@@ -226,63 +225,12 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
...
     label_map = {label : i for i, label in enumerate(label_list)}
 
-    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id) for example in examples]
+    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id, sep_token_extra) for example in examples]
 
     with Pool(process_count) as p:
         features = list(tqdm(p.imap(convert_example_to_feature, examples, chunksize=100), total=len(examples)))
 
     return features_length - len(input_ids)
-    if pad_on_left:
```
Comment (julian-pani) [inline utils.py]: This whole thing looks like a bad merge or copy/paste

File: utils.py [hunk 3]
```diff
@@ -226,63 +225,12 @@ def convert_examples_to_features(examples, label_list, max_seq_length,
 
     label_map = {label : i for i, label in enumerate(label_list)}
 
-    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id) for example in examples]
+    examples = [(example, label_map, max_seq_length, tokenizer, output_mode, cls_token_at_end, cls_token, sep_token, cls_token_segment_id, pad_on_left, pad_token_segment_id, sep_token_extra) for example in examples]
```
Comment (julian-pani) [inline utils.py]: Some changes existed in one of the `convert_examples_to_features`, some changes existed in the other.

I merged the changes:
* Adding `sep_token_extra=False` arg
* Passing `sep_token_extra` to in `examples` list.
* Adding `process_count=cpu_count() - 2` arg
* `chunksize = 500`

File: utils.py (hunk 2 above)
Comment (ThilinaRajapakse) [reply utils.py]: Yeah, it looks like the code from `convert_example_to_feature` function somehow got copied into the `convert_examples_to_features` function (as a duplicate to boot). Thanks, I'll merge this.

**Review by ThilinaRajapakse — APPROVED** [review]

**Pull request merged**